avocado/core/mux.py

```diff
@@ -109,26 +109,21 @@ def update_defaults(self, defaults):
         combination.merge(self.root)
         self.variants = MuxTree(combination)
 
-    def str_variants(self):
+    def str_variants(self, summary=False, contents=False):
```

Comment:
Btw you keep pointing to some unittests you wrote but I don't see them anywhere... As I mentioned earlier I intend to extend the unittests, but at this point I didn't even know whether this approach would be acceptable or not.